spec/lib/active_merchant/billing/klarna_gateway_spec.rb

```diff
@@ -180,19 +183,5 @@
         expect { create_profile }.not_to change(payment_source.reload, :customer_token).from(initial_customer_token)
       end
     end
-
-    context 'when Spree::Order#klarna_tokenizable? is false' do
```

Comment:
Why did you remove this test?

Reply:
I didn't remove it.

It got removed when I forked it and copied over the changes I had made.
I am looking for any more changes to the files I have modified right now.
Will add it again

Reply:
oh, this is pretty dangerous. probably you have forked the repo before the PR that adds these changes was merged. Please update your master and rebase this branch again 🙏